test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestManagementFactory.java

```diff
@@ -833,7 +833,8 @@ public void testMemoryPoolMXBeanProxy() {
 			MemoryUsage directUsage = realBean.getCollectionUsage();
 			validateMemoryUsage(proxyUsage);
 			validateMemoryUsage(directUsage);
-			AssertJUnit.assertEquals(directUsage, proxyUsage);
+			/* directUsage and proxyUsage could be different, if gc happened between getCollectionUsage() calls*/
+//			AssertJUnit.assertEquals(directUsage, proxyUsage);
 
```

Comment:
Are you keeping it in the code as a comment to come back to it?  If it is invalid, do you not simply want to remove it?

Reply:
removed the assertion check and leave comment for a reminder